python/docs/source/conf.py

```diff
@@ -69,6 +69,7 @@
     "github_repo": "kaskada",
     "github_version": "main",
     "doc_path": "kaskada/docs/source",
+    "google_analytics": "G-HR9E2E6TG4",
```

Comment:
We may also want this to be an environment variable, so that it is only on in the deployed docs?

Reply:
Let's hard code it for now - it's what we're doing in a couple other places.